freqtrade/strategy/resolver.py

```diff
@@ -79,6 +82,22 @@ def _load_strategy(
             # Add extra strategy directory on top of search paths
             abs_paths.insert(0, extra_dir)
 
+        if ":" in strategy_name:
+            logger.debug("loading base64 endocded strategy")
```

Comment:
i think this should be info, not debug (assuming it's only called once ... ?)

Reply:
only called once, but not sure if people actually have to be aware of it. Up to you

Reply:
i think so, as it can create misconception if i think i load via `-s` ... but have a strategy in the configuration file ... (at least i think that's a possibility)